ldms/python/ldmsd/ldmsd_communicator.py

```diff
@@ -616,6 +617,7 @@ class LDMSD_Request(object):
     SET_SEC_MOD = 0x600 + 19
     LOG_STATUS = 0x600 + 20
     STATS_RESET = 0x600 + 21
+    PROFILING = 0x600 + 31
```

Comment:
@nichamon, is there some reasons we are bumping by 31 instead of 22?

Reply:
> @nichamon, is there some reasons we are bumping by 31 instead of 22?

It's the same number defined in [ldmsd_request.h:168](https://github.com/nichamon/ovis/blob/a77173a37fbd456183f6095f4dd0121532986ea7/ldms/src/ldmsd/ldmsd_request.h#L168) in this change.

Reply:
@tom95858 I added the comment to explain why the ID skips the range.